src/openzaak/components/zaken/models/zaken.py

```diff
@@ -74,10 +74,41 @@ class Zaak(ETagMixin, AuditTrailMixin, APIMixin, models.Model):
     bewaakt moeten worden.
     """
 
+    # acts as the primary key - the (data) migrations ensure that the references
+    # are correct
+    identificatie_ptr = models.OneToOneField(
+        ZaakIdentificatie,
+        on_delete=models.PROTECT,
+        parent_link=True,
+        primary_key=True,
```

Comment:
it will not be generated indeed, instead the `ZaakSerializer` will assign/set on the `Zaak` instance, and django internally manages the multi-table inheritance - it will create the `ZaakIdentificatie` as needed. There is an existing test for that case: `openzaak.components.zaken.tests.test_create.CreateZaakTests.test_identificatie_all_characters_allowed`